FormattingStyleGuide.md

```diff
@@ -65,6 +65,10 @@ This document provides formatting and style conventions for publishing documents
 - Use programmer's editor (e.g. [Sublime Text](https://www.sublimetext.com/)), or just Notepad to convert documents from Word, Confluence, PDF a.o. formats to Markdown.
 - Specialized converter software (e.g. [Pandoc](http://pandoc.org/)) can be used, but will rarely justify the learning effort.
 
+## Drawing tool
+- Use draw.io to create drawings, flowcharts and pictures for X-Road documentation. Draw.io (https://www.draw.io/) is completely free online diagram editor. It can be also used in offline mode by going to the following URL into your web browser: https://www.draw.io/app or by using Chrome Application.
+- Use PNG file format in documents and save the original source file as native XML. Both files must be committed into Github to very same location where the document exists.
```

Comment:
We usually put images into "img" sub-folder not to the same location as document. Should we mention "img" folder, or rather should get rid of "img" folders in the future?